### DISCUSSION TITLE
Any way to find MAC address besides the website?

### DISCUSSION THREAD

**Discussion by malvarenga123** [darthcloud/BlueRetro#980] [Q&A] [answered]
Hi, every time I try to connect my adapter using the website, I only get "NotFoundError", but I am able to update it using Flash Download Tool and I can see debug info by connecting to it using Putty.
Is there any other way I can find its MAC address, so I can pair my PS3 controllers?
My device is the RetroScaler Mini for PS1/PS2.

**Comment by darthcloud** [comment, marked as answer]
You can get it from the log too
https://github.com/darthcloud/BlueRetro/wiki/Getting-BlueRetro-debug-logs-via-Serial-port-Windows-10

look for a line that got `local_bdaddr` in it.

**Reply by darthcloud** [reply]
Some RetroScaler adapter don't support the BlueRetro log baud rate, so it's possible it's not possible to get it.

**Reply by malvarenga123** [reply]
Thanks! That did the trick.

**Comment by darthcloud** [comment]
Also for the web you could try on another laptop.
